main.py

```diff
@@ -1,10 +1,44 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        data = json.load(file)
+
+    for player_name, info in data.items():
+        if not Race.objects.filter(name=info["race"]["name"]).exists():
+            Race.objects.create(name=info["race"]["name"],
+                                description=info["race"]["description"])
+        current_race = Race.objects.get(name=info["race"]["name"])
+
+        if info["guild"]:
+            if not Guild.objects.filter(name=info["guild"]["name"]).exists():
+                Guild.objects.create(name=info["guild"]["name"],
+                                     description=info["guild"]["description"])
+            current_guild = Guild.objects.get(name=info["guild"]["name"])
```

Comment:
@Nattalli, can you please tell why `get_or_create()` is not in the readme, if you say that it's the best practice to use it? Maybe it makes sense to add it to the following note?
> Note: You can check, if some record already exists using `Model.objects.filter(some_field="data").exists()` (returns `True` or `False`)

Or is it not there to encourage students to do their own research? :)

Reply:
+, because it should be in the next lessons. Therefore, it is not necessary to use it now, but if someone is interested and is not too lazy to find out about this method, then I point out that it exists and it optimizes the code at least by making it more readable

Reply:
Got it. Thanks for your answer!